extension.yaml

```diff
@@ -22,7 +22,7 @@ version: 0.1.0
 specVersion: v1beta
 
 # Friendly display name for your extension (~3-5 words)
-displayName: Search in Firestore with MeiliSearch
+displayName: Search in Firestore with Meilisearch
```

Comment:
```suggestion
displayName: Search in your Firestore content with Meilisearch
```

Reply:
This will be the name displayed when you search and install your extension. So I think that something very simple is maybe better no? 
![Screenshot 2022-03-16 at 16 31 55](https://user-images.githubusercontent.com/16155041/158627707-1d26e3fe-d21a-4f7b-b16f-537e230d7d80.png)

Reply:
Okey!